### ISSUE TITLE
Move Prefer RGB to switch from configuration

### ISSUE THREAD

**Issue by broyuken** [basnijholt/adaptive-lighting#74]
Looking to do something similar to https://github.com/basnijholt/adaptive-lighting/issues/36. Could you please consider moving the prefer rgb over color temp out of the configuration and into a switch? That way I could toggle it off and on along with my sleep mode. I have some bulbs that are too bright even at 1% when using color temp but when I use rgb they can go nice and low. I like the color temps during the day but would like my "night light" to be super dim by changing from color temp to rgb.

**Comment by broyuken** [comment]
I have attempted to use the following service call, and the prefer_rgb_color works. But only until the light checks in again, then it goes back to false. How do I make this change permanent, or at least until I turn the switch off and on again? 

```
service: adaptive_lighting.apply
data:
  entity_id: switch.adaptive_lighting_sylvania_bulbs
  lights: light.office_lamp_floor
  prefer_rgb_color: true
```

**Comment by broyuken** [comment]
You have a number of requests on here to be able to do prefer rgb at night, but not during the day. Could you please give us the option to toggle this the same way that we toggle adjust color and brightness by exposing switches? It seems that adaptive_lighting.apply is only a one time change, and doesn't change the config permanently. The workaround is to have multiple instances with automations to turn one or the other off which is extremely sloppy and does not scale. Please please please give us this option as a switch @basnijholt .

**Comment by RubenKelevra** [comment]
Due to a feature freeze of v1 the milestone has been changed to v2.

Keep this ticket open until you can verify that it's properly working in the upcoming version 2.

**Comment by basnijholt** [comment]
Who's excited?!

<img width="498" alt="image" src="https://user-images.githubusercontent.com/6897215/187840527-f0c88d7f-541f-4f73-bac9-a3d1ed35892c.png">

See https://github.com/basnijholt/adaptive-lighting/pull/329

**Comment by basnijholt** [comment]
I have just released [1.1.0](https://github.com/basnijholt/adaptive-lighting/releases/tag/1.1.0) where you can set the sleep color using RGB.

This issue can finally be closed! :tada

**Comment by broyuken** [comment]
🙌🏻🙌🏻🙌🏻
